spec/api/v1/assessments_api_spec.rb

```diff
@@ -24,22 +24,6 @@
     end
   end
 
-  describe 'GET #problems' do
```

Comment:
why was this test case removed?

Reply:
Was having issues with it, but those are resolved so will add it back to `problems_api_spec.rb`